.gitignore

```diff
@@ -31,3 +31,5 @@ docs-prerelease.json
 docs.json
 .dub/
 dpl-docs/dpl-docs
+
+/d.tag
```

Comment:
I'd rather use $TMP for junk instead of having it there and ignoring it. (That would be nice for the LaTeX-related stuff, too.)